packages/ui/src/ui/store/actions/navigation/modals/create-directory.tsx

```diff
@@ -11,13 +11,10 @@ import {
     CREATE_DIRECTORY,
     CREATE_MESSAGE,
 } from '../../../../constants/navigation/modals/create-directory';
-import {
-    hideError,
-    showErrorInModal,
-} from '../../../../store/actions/navigation/modals/path-editing-popup';
+import {hideError, showErrorInModal} from './path-editing-popup';
 import {HIDE_ERROR} from '../../../../constants/navigation/modals/path-editing-popup';
 import CancelHelper from '../../../../utils/cancel-helper';
-import {RootState} from '../../../../store/reducers';
+import {RootState} from '../../../reducers';
```

Comment:
All the changes looks unnecessary in this file. Were the changes forgotten to remove?

Reply:
ide suggested a shorter route. There's nothing wrong with fixing it